### DISCUSSION TITLE
No access possible via login mask

### DISCUSSION THREAD

**Discussion by Cardellino** [linuxdeepin/developer-center#3523] [Q&A | 问答板块] [answered]
Hello, yesterday I had an update on my Arch Linux system. Today I start Deepin and stay on the mask.
Access not possible....
Does anyone have this error after the update?
See photo.
![IMG_20221220_152139](https://user-images.githubusercontent.com/73848158/208696277-90b184b3-e7a5-4822-993e-fad51c355841.jpg)

**Comment by Cardellino** [comment]
update from yesterday 19.12.2022
------
deepin-launcher
deepin-image-editor
deepin-gettext-tools
deepin-editor
deepin-draw
deepin-dock
deepin-daemon
deepin-control-center
deepin-desktop-schemas
deepin-camera
deepin-album

**Comment by BLumia** [comment]
Can reproduce. Downgrade `deepin-desktop-schemas` seems to work on my side. cc @felixonmars 

edit: Downgrade `deepin-desktop-schemas` to 5.10.11-1 can make the login screen work again but still have some issue. Will let you know if there are any progress on this issue.

**Reply by Cardellino** [reply]
Hello BLumia, I downgraded to version deepin-desktop-schemas-5.10.6-2.
It takes a while until the login mask is visible. 
then I notice that the docker bar is not visible.
I have to start all programs from the terminal.
I don't know if it's just the desktop scheme.
When starting the system, it takes a while until the login mask is displayed....
Ok, I'll wait for it to be fixed..
Thanks for your support...

**Reply by BLumia** [reply]
Yes, you can install another DE as a workaround before the package issue gets fixed (you can select DE from the bottom-right side of the login screen, click the button near-by the word `x11` and you'll be able to select another DE.

**Reply by Cardellino** [reply]
Hello BLumia, I could only fix the error by downgrading -> sudo pacman -U deepin-daemon-5.14.105-1-x86_64.pkg.tar.zst.
Registration mask available immediately. Dock can also be seen again.
![Bildschirmaufnahme_Auswahlbereich_20221222103053](https://user-images.githubusercontent.com/73848158/209103627-f909b68e-3d95-4ade-835a-acae770e4d8f.png)

What doesn't work is the starter. Opens and closes again.
Many greetings

**Comment by felixonmars** [comment, marked as answer]
Please try again with deepin-daemon 5.14.106-2 with a patch applied for https://github.com/godbus/dbus/pull/348

I don't get the login screen issue though, hopefully someone else would figure out the problem.

**Reply by BLumia** [reply]
After upgrading to `deepin-daemon 5.14.106-2` the issue was gone. The login screen issue no longer exists on my side too.

**Reply by Cardellino** [reply]
After upgrading to Deepin daemon 5.14.106-2 the problem was solved.
What does not work is the STARTER!
When you click, it opens and immediately closes again.
And that also depends on the update from 19.12.2022

**Reply by BLumia** [reply]
> STARTER

Do you mean the launcher (the button at the bottom-left side)? It seems it will only happen the first time you click it. If you click it again then it will work as intended (at least on my machine).

**Reply by Cardellino** [reply]
Yes, I forgot BLumia that it is about the launcher (starter). Excuse me....
When I click on it then flickers and closes again. The launcher does not remain open.
Before the update on December 19, this had worked without any problems.

**Reply by Cardellino** [reply]
The video. The launcher does not remain open.

https://user-images.githubusercontent.com/73848158/209376228-14ec2fe8-a5fb-4b79-b1a9-3d8128b1783b.mp4

**Reply by BLumia** [reply]
I'm using the full screen launcher and it works fine so far. Will test it locally later.

Since it's probably a different issue than the login mask one. Maybe we can start a new discussion thread and discuss the issue there. So once the new issue gets resolved, we can mark the appropriate solution as answer so it could be helpful for other people who also have similar issue :)

**Reply by Cardellino** [reply]
Hello BLumia, after 10 times click on the launcher it worked and have it on -> full screen launcher. After that the same problem!
OK. The ticket can be closed. Will open a new one for the launcher.
Thanks for your support.

https://user-images.githubusercontent.com/73848158/209467494-e6aa9d03-ed0b-4031-803a-5079d7e7ebf0.mp4
